org.opentosca.planbuilder/org.opentosca.planbuilder.postphase.plugin.instancedata/src/main/java/org/opentosca/planbuilder/postphase/plugin/instancedata/bpel/InstanceStates.java

```diff
@@ -98,7 +105,7 @@ public static String getNextStableOperationState(final String state) {
             // given state is stable
             switch (state) {
                 case "INITIAL":
-                    return "CREATED";
+                    return "CREATED";  // FIXME: unreachable, because operationPostStates doesn't contain value "INITIAL"
```

Comment:
Is this still an issue?

Reply:
It's still unreachable, but it doesn't seem to cause issues.